features/steps/formatting_steps.rb

```diff
@@ -251,7 +251,7 @@
 end
 
 Then(/^I should see a red error message$/) do
-  run_output.should include(red("⌦  " + SAMPLE_PODS_ERROR.gsub('error: ', '')))
+  run_output.should include(red("❌  " + SAMPLE_PODS_ERROR.gsub('error: ', '')))
```

Comment:
I like it that compilation errors are different from test failures, but that’s really a matter of taste. If you prefer red("✗") instead of ❌ I’ll happily change it. Both ✗ and ❌ are better than ⌦ IMHO.

Unfortunately, the symbol Xcode is using is not available in unicode. Here are a few more alternative symbols:
* ❗️ (U+2757 HEAVY EXCLAMATION MARK SYMBOL)
* ‼️ (U+203C DOUBLE EXCLAMATION MARK)
* 🚫 (U+1F6AB NO ENTRY SIGN)
* ⛔️ (U+26D4 NO ENTRY)

My preference goes to ❌.